src/components/events/components/EmailListCard.jsx

```diff
@@ -0,0 +1,125 @@
+/** @jsxImportSource @emotion/react */
+import "twin.macro";
+import React, { useState } from "react";
+import { IdIcon, MailIcon } from "../../icons/Icons";
+import { GenericInput } from "../../form/GenericInput";
+
+
+export const EmailListCard = () => {
+  const [status, setStatus] = useState(null);
+  const [email, setEmail] = useState("");
+  const [name, setName] = useState("");
+  const FORM_URL = `https://api.convertkit.com/v3/forms/${process.env.REACT_APP_CONVERTKIT_FORM_ID}/subscribe`;
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const data = new FormData();
+    data.append('api_key', process.env.REACT_APP_CONVERTKIT_API_KEY);
+    data.append('first_name', name);
+    data.append('email', email);
+
+    try {
+      const response = await fetch(FORM_URL, {
```

Comment:
nit: this should follow the standard of doing this with axios but we can fix that later